examples/fix/simple/.expected/config.yaml

```diff
@@ -0,0 +1,4 @@
+runCount: 1
+testType: eval
+image: gcr.io/kpt-fn/fix:unstable
```

Comment:
do we want the name to be just `fix`? Given it's for niche use case of migration, we may want to choose a more descriptive name.

Reply:
We used this name for same functionality in v0.X. User are familiar with it. Kustomize also uses same command name for migrating.